src/style.css

```diff
@@ -0,0 +1,5 @@
+@import url(./normalize.css);
+@import url(./stars.css);
+@import url(./card.css);
+
```

Comment:
Not a mistake but recomendation to keep your css files for blocks in another folder. And for all css files create folder like 'styles'